js-miniapp-bridge/src/fetch/response.ts

```diff
@@ -0,0 +1,50 @@
+import { NativeFetchResponse, FetchResponse } from '../types/fetch';
+
+export class DecodedFetchResponse implements FetchResponse {
+  private encoding = 'utf-8';
+  headers: Record<string, string>;
+  ok: boolean;
+  status: number;
+  statusText: string;
+  url: string;
+  body: Uint8Array;
```

Comment:
Why do we have to declare again when `Response` and `ResponeBody` have them already?

Reply:
since DecodedFetchResponse extends the Response type, it's required to implement all of its parent properties in its class.